test/ui/test_spec.js

```diff
@@ -88,7 +88,9 @@ describe('UI tests', function() {
               });
           }, 10000);
         })
-        .finally(() => done());
+        .then(driver.executeScript("window.datalab.vminfo.vm_name='fake_vm_name_for_testing';" +
+              "require(['static/appbar'],appbar=>appbar.setAccountMenuVminfo());"))
```

Comment:
this should probably go in a `beforeAll` block though, we might have other tests that rely on it.